detectable/src/main/java/com/synopsys/integration/detectable/detectables/pipenv/parse/model/PipfileLock.java

```diff
@@ -0,0 +1,13 @@
+package com.synopsys.integration.detectable.detectables.pipenv.parse.model;
```

Comment:
Put classes that are directly serializable by a library in a package called `data` instead of `model`